### PR TITLE
AI mapping

### PR DESCRIPTION
**Pull request by Gegy**

### CODE REVIEW

**Comment by liach** [comment]
can we rename pathing to.navigation

File: mappings/net/minecraft/entity/mob/MobEntity.mapping [hunk 1]
```diff
@@ -68,6 +68,7 @@ CLASS aho net/minecraft/entity/mob/MobEntity
 		ARG 0 stack
 	METHOD f setTarget (Lahn;)V
 	METHOD n setCanPickUpLoot (Z)V
+	METHOD o addGoals ()V
```
Comment (Pyrofab) [inline mappings/net/minecraft/entity/mob/MobEntity.mapping]: Why not `initGoals` ?

Comment (Gegy) [reply mappings/net/minecraft/entity/mob/MobEntity.mapping]: They both work the same, imo. Any reason for choosing one over the other?

Comment (Pyrofab) [reply mappings/net/minecraft/entity/mob/MobEntity.mapping]: We already have `initAttributes` and `initDataTracker`, so in my opinion that method should be named similarly.

Comment (Gegy) [reply mappings/net/minecraft/entity/mob/MobEntity.mapping]: Good point, will change. 👍

**Review by modmuss50 — APPROVED** [review]

**Comment by modmuss50** [comment]
Updating, merging, open an issue if you have further comments.

**Pull request merged**